src/components/pages/Profile.tsx

```diff
@@ -1,14 +1,26 @@
+import { TezosToolkit } from "@taquito/taquito";
 import React from "react";
 import { useParams } from "react-router-dom";
 import DefaultProfilePic from "../../assets/default-user.png";
-import { HoundInfo, HoundRarity } from "../../scripts/hound-genome";
+import { HoundInfo} from "../../scripts/hound-genome";
 import "../css/Profile.css";
 import HoundCard from "../HoundCard";
+import UpdateContract from "../UpdateContract";
 
 // Wallet addresses can be too long for us to render them fully, so
 // we only render a substring.
 const MaxAddressLen = 12;
 
+type TezosProps = {
+  Tezos: TezosToolkit;
+  hounds: HoundInfo[];
+};
+
+type Hound = {
```

Comment:
Why do we need this wrapper type? Also what do you think about the name? I find it a bit confusing that there' `Hound` and `HoundInfo`, which mean different things.

Reply:
This was due to some weird error thrown by react, I didn't format stuff properly and I am not too sure why that error is thrown so I could probably ask that on the call tomorrow.